src/main/webapp/creatives/maps/simid_map.css

```diff
@@ -1,6 +1,26 @@
 /* Styling for maps creative */ 
 
 /*Styling for initial state */
+button {
+    background: rgba(0,0,0,0.5);
+    border: 1px solid rgba(255,255,255,0.5);
+    border-right: 0;
+    box-sizing: content-box;
+    color: #fff;
+    font-size: 18px;
+    line-height: normal;
+    min-width: 0;
+    padding: 10px 6px 8px 10px;
+    width: auto;
+    text-align: center;
+    transition: opacity .5s cubic-bezier(0.0,0.0,0.2,1);
+    text-shadow: 0 0 4px rgba(0,0,0,0.75);
+}
+
+button:hover {
```

Comment:
Nice! Another good thing to add, would be some sort of highlight when the button is "focused" (it might already happen by default, but good to make sure that the default highlight looks good on your creative).  Focus highlights are super useful for accessibility purposes.